playground/TestShop/TestShop.AppHost/Program.cs

```diff
@@ -89,7 +94,79 @@
 // dashboard launch experience, Refer to Directory.Build.props for the path to
 // the dashboard binary (defaults to the Aspire.Dashboard bin output in the
 // artifacts dir).
-builder.AddProject<Projects.Aspire_Dashboard>(KnownResourceNames.AspireDashboard);
+builder.AddProject<Projects.Aspire_Dashboard>(KnownResourceNames.AspireDashboard)
```

Comment:
nit: probably come up with another use case rather than a command on the dashboard resource?